src/utils/mockDataUtils.ts

```diff
@@ -389,3 +392,57 @@ const generateNodeArray = <T extends (...args: any) => any>(
 ): ReturnType<T>[] => {
   return Array.from({ length }).map((_, i) => fakeFunc(i));
 };
+
+export enum StudyLevel {
+  Preschool = 'PRESCHOOL',
+  Grade_1 = 'GRADE_1',
+  Grade_2 = 'GRADE_2',
+  Grade_3 = 'GRADE_3',
+  Grade_4 = 'GRADE_4',
+  Grade_5 = 'GRADE_5',
+  Grade_6 = 'GRADE_6',
+  Grade_7 = 'GRADE_7',
+  Grade_8 = 'GRADE_8',
+  Grade_9 = 'GRADE_9',
+  Grade_10 = 'GRADE_10',
+  Secondary = 'SECONDARY',
+}
+
+export const initialStudyLevels = Object.keys(StudyLevel).map(
+  (lvl: string) => StudyLevel[lvl as keyof typeof StudyLevel] as StudyLevel
+);
```

Comment:
Would this works the same way here?

`export const initialStudyLevels = Object.values(StudyLevel);`

Reply:
Definitely! I don't know how I came up with this solution :D Maybe an editor gave a hint.